policy/examples/requirements.md

```diff
@@ -0,0 +1,717 @@
+# Policy Requirement Examples
+
+This file presents a series of example [Requirements](../README.md#requirement) documents for Agencies to use as templates.
+
+## Table of Contents
+
+- [Policy and Geography](#policy-and-geography)
+- [Vehicles Only](#vehicles-only)
+- [Trips Only](#trips-only)
+- [Trips with No Routes, Vehicles IDs, or Dates](#trips-with-no-routes-vehicle-ids-or-dates)
+- [Provider and Other APIs](#provider-and-other-apis)
+- [Agency](#agency)
+- [Geography Driven Events](#geography-driven-events)
+- [GBFS Only](#gbfs-only)
+
+## Policy and Geography
+
+Version 1.2.0 of MDS Policy and Geography for agencies to publish rules/fees/incentives and operating/equity/no-ride/slow speed/parking areas to all providers, and require GBFS's geofencing_zones.  
+
+```json
+{
+  "metadata": {
+    "mds_release": "1.2.0",
+    "file_version": "4",
+    "last_updated": "1611729218",
+    "max_update_interval": "T1M",
```

Comment:
I'm pretty sure this should be `P1M` (if the update interval should be monthly), or `PT1M` (if the update interval should be minutely). I don't think that `T1M` is a valid ISO8601 duration.

Ditto for other occurrences of `T1M` in the examples.

Reply:
Yes it was correct in the spec definition example, but not correct in all the examples. Updated them all now to be P1M.